misaligned/misaligned.cs

```diff
@@ -3,13 +3,13 @@
 
 namespace MisalignedSpace {
     class Misaligned {
+        static string[] majorColors = {"White", "Red", "Black", "Yellow", "Violet"};
+        static string[] minorColors = {"Blue", "Orange", "Green", "Brown", "Slate"};
         static int printColorMap() {
-            string[] majorColors = {"White", "Red", "Black", "Yellow", "Violet"};
-            string[] minorColors = {"Blue", "Orange", "Green", "Brown", "Slate"};
             int i = 0, j = 0;
-            for(i = 0; i < 5; i++) {
-                for(j = 0; j < 5; j++) {
-                    Console.WriteLine("{0} | {1} | {2}", i * 5 + j, majorColors[i], minorColors[i]);
+            for(i = 0; i < majorColors.Length; i++) {
+                for(j = 0; j < minorColors.Length; j++) {
+                    Console.WriteLine("{0} | {1} | {2}", i * 5 + j + 1, majorColors[i], minorColors[j]);
                 }
             }
             return i * j;

```

Comment:
The number of color pairs is verified. But how do we verify if the formatting was done as required? Can you come up with a method to verify and validate the formatted content?